src/DotNet.Status.Web/Controllers/GitHubHookController.cs

```diff
@@ -53,22 +59,187 @@ private async Task EnsureLabelsAsync()
             );
         }
 
+        private readonly SimpleJsonSerializer _serializer = new SimpleJsonSerializer();
+
+        private async Task<T> DeserializeGitHubWebHook<T>()
+        {
+            // Octokit has a serializer and types built for github rest apis, use them
+            string data;
+            using (var reader = new StreamReader(Request.Body))
+            {
+                data = await reader.ReadToEndAsync();
+            }
+
+            var payload = _serializer.Deserialize<T>(data);
+            return payload;
+        }
+
+        [GitHubWebHook(EventName = "pull_request_review_comment")]
```

Comment:
I'm highly skeptical our poor little basic web app is going to be able to handle this.  The number of comments posted in all dotnet repositories is large. Really large.

Reply:
I looked at the data in CloudMine.

Over the last 30 days for the dotnet GitHub org, there was a peak of 32 PullRequestReviewComments per second, minimum 1 and average 1.14.

Reply:
Not surprisingly, almost all are action 'created' so will take the more expensive path.

Reply:
App Insights has current requests per second over the last 30 days to this site at max 29, min 1, average 2.

So... rough estimate of increased load?

Reply:
Oh, looks like Issue activity is even higher at 60 per second max, 1 min, 1.1 avg.

Reply:
The current request rate is negligible, so it's not really a worry about "increased load". Essentially our current load is "zero". :-)  So it's just "can a free teir web app in Azure handle the comment load". At 1 a second, it looks like.... probably?